### PR TITLE
feat: added cards component

### PR DESCRIPTION
**Pull request by kedark152**
### feat: Added Cards Components:
Re-usable classes for Cards are written in cards.css file.
cards.css is then imported in docs.css file just like all the components to display.

**Features:**
1. Cards with badges & dismiss. (vertical cards)
2. Cards with text overlay & Text Only Card.
3. Two Horizontal Cards.
4. All Cards gets shadow on Hovering on it.

Watch the Preview: https://deploy-preview-3--forest-ui.netlify.app/docs-site/docs

### CODE REVIEW

File: stylesheets/components/cards.css [hunk 1]
```diff
@@ -0,0 +1,172 @@
...
+    border-radius: 0.375rem;
+    z-index: 2;
+    cursor: context-menu;
+}
+
+.card-overlay-text{
+    position: absolute;
+    top: 50%;
+    left: 50%;
+    background-color: #dee2e6;
```
Comment (lannisterrr) [inline stylesheets/components/cards.css]: Try to use variables for colors as much as possible.

**Review by lannisterrr — COMMENTED** [review]
Other than point mentioned above , everything seems fine.

File: stylesheets/components/cards.css (hunk 1 above)
Comment (kedark152) [reply stylesheets/components/cards.css]: Thanks Siddhant!! made suggested changes on line 125. Please check & approve my PR.

File: docs-site/docs.html [hunk 2]
```diff
@@ -145,39 +145,242 @@ <h3>Badge on Icons</h3>
     <hr />
     <!-- Buttons -->
     <h2>Buttons</h2>
```
Comment (Bhakti199) [inline docs-site/docs.html:147]: here you have already given heading `h2` so no need to add comments

**Review by Bhakti199 — COMMENTED** [review]
Overall code is nicely written and structured, Also like that you are using proper naming conventions and constant rem units throughout the code.

File: docs-site/docs.html (hunk 2 above)
Comment (kedark152) [reply docs-site/docs.html:147]: Hi Bhakti, Thanks for your Review. 
At the beginning of each component code section, I write component name in the comment at start, as it acts like divider between multiple code components and helps me for easily reading the code.
Request you to please provide approval.

**Review by Bhakti199 — APPROVED** [review]

**Review by lannisterrr — COMMENTED** [review]
Everything seems fine

**Review by lannisterrr — APPROVED** [review]

**Pull request merged**